tests/schema_test.py

```diff
@@ -154,3 +154,29 @@ def test_check_for_cycles_negative(self):
 
         with self.assertRaisesRegexp(AssertionError, 'cycle detected'):
             check_for_cycles(schemata)
+
+    def test_links_to_annotation(self):
+        special_node = [
+            'annotation', 'clinical', 'experimental_strategy', 'metaschema',
+            'platform', 'program', 'project', 'publication', 'root', 'tag'
+        ]
```

Comment:
It's a minor thing but if we use a set here, the `x in set` that we do in `_is_excluded_node` will be constant time instead of linear.